modules/preview-react/information-highlight/lib/Icon.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {SystemIcon, SystemIconProps} from '@workday/canvas-kit-react/icon';
+import {createSubcomponent} from '@workday/canvas-kit-react/common';
+import {cssVar} from '@workday/canvas-kit-styling';
+import {
+  infoIcon,
+  exclamationCircleIcon,
+  exclamationTriangleIcon,
+} from '@workday/canvas-system-icons-web';
+import {base} from '@workday/canvas-tokens-web';
+import {useInformationHighlightModel} from './modelHook';
+
+type Variant = 'informational' | 'caution' | 'attention';
+
+const iconStyles = {
```

Comment:
```suggestion
const defaultIconStyles = {
```

to match `defaultIcons`

Reply:
oohh changed!